packages/webdriverio/src/commands/element/click.ts

```diff
@@ -75,14 +86,25 @@ const log = logger('webdriver')
     })
  * </example>
  *
+ * <example>
+    :longpress.example.js
+    it('should be able to open the contacts menu on iOS by executing a longPress', async () => {
+        const contacts = await $('~Contacts')
```

Comment:
$ doesn't have to be awaited
```suggestion
        const contacts = $('~Contacts')
```

Reply:
Hi @erwinheitzman 
Thanks, I'm following the docs here, this doc has the `await` added for all elements